### PR TITLE
Added examples for jdbc integration

### PR DESCRIPTION
**Pull request by zaleslaw**

### CODE REVIEW

File: examples/idea-examples/imdb/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    kotlin("jvm")
+    id("org.jetbrains.kotlinx.dataframe")
+}
+
+repositories {
+    mavenLocal() // in case of local dataframe development
+    mavenCentral()
+}
+
+kotlin.sourceSets.getByName("main").kotlin.srcDir("build/generated/ksp/main/kotlin/")
```
Comment (koperagen) [inline examples/idea-examples/imdb/build.gradle.kts:14]: Does it resolve & compile without this line?

File: examples/idea-examples/imdb/src/main/kotlin/org/jetbrains/kotlinx/dataframe/examples/jdbc/Example_1_Define_schema_manually.kt [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    val df = DataFrame.readSqlTable(dbConfig, TABLE_NAME_MOVIES, 10000).cast<Movies>()
+
+    // print the dataframe
+    df.print()
+
+    // print the dataframe metadata and statistics
+    df.describe().print()
+
+    // print names of top-10 rated films
+    df.sortByDesc { it[Movies::rank] }
```
Comment (koperagen) [inline examples/idea-examples/imdb/src/main/kotlin/org/jetbrains/kotlinx/dataframe/examples/jdbc/Example_1_Define_schema_manually.kt:36]: Strange that you use KProperties API here and not extension properties

File: examples/idea-examples/imdb/src/main/kotlin/org/jetbrains/kotlinx/dataframe/examples/jdbc/Example_1_Define_schema_manually.kt [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    val year: String
+    val rank: String
+}
+
+fun main() {
+    // define the database configuration
+    val dbConfig = DatabaseConfiguration(URL, USER_NAME, PASSWORD)
+
+    // read the table
+    val df = DataFrame.readSqlTable(dbConfig, TABLE_NAME_MOVIES, 10000).cast<Movies>()
```
Comment (koperagen) [inline examples/idea-examples/imdb/src/main/kotlin/org/jetbrains/kotlinx/dataframe/examples/jdbc/Example_1_Define_schema_manually.kt:27]: I wonder.. `cast<Movies>()` isn't very safe, although we use it some times.  Let's use `cast<Movies>(verify=true)` in examples just to be sure and provide clear error and not some sudden NPE / class cast exceptions

**Review by koperagen — CHANGES_REQUESTED** [review]
A lot of code :slightly_smiling_face: It looks interesting, i want to read it carefully. For now left comment on simple things that draw my attention

File: examples/idea-examples/imdb/src/main/kotlin/org/jetbrains/kotlinx/dataframe/examples/jdbc/Example_1_Define_schema_manually.kt (hunk 2 above)
Comment (zaleslaw) [reply examples/idea-examples/imdb/src/main/kotlin/org/jetbrains/kotlinx/dataframe/examples/jdbc/Example_1_Define_schema_manually.kt:36]: It does not work correctly in the one project and it was a reason, but now it works correctly, moved to extension

File: examples/idea-examples/imdb/src/main/kotlin/org/jetbrains/kotlinx/dataframe/examples/jdbc/Example_1_Define_schema_manually.kt (hunk 3 above)
Comment (zaleslaw) [reply examples/idea-examples/imdb/src/main/kotlin/org/jetbrains/kotlinx/dataframe/examples/jdbc/Example_1_Define_schema_manually.kt:27]: probably better to make verify=true - default option, if it's so important, what do you think? Is there another way to cast raw result to the schema?

File: examples/idea-examples/imdb/build.gradle.kts (hunk 1 above)
Comment (zaleslaw) [reply examples/idea-examples/imdb/build.gradle.kts:14]: Better to create an issue for all examples (it's the same in all)

**Comment by zaleslaw** [comment]
I decided to move it for a while into a separate repository

**Pull request closed without merging**